Fixes #10446, shorten "out of element" message

yozaam opened the pull request:
Based on the suggestion this has been made into  "element end"

<!--
Please fill in the following template, for an explanation of the sections see:
https://github.com/nvaccess/nvda/wiki/Github-pull-request-template-explanation-and-examples
-->

### Link to issue number:
 #10446 

### Summary of the issue:
Instead of "out of list", announce 'list end", since this is shorter

### Description of how this pull request fixes the issue: 
Have changed the strings where we have the longer message

### Testing performed:

### Known issues with pull request:

### Change log entry:

Section: New features, Changes, Bug fixes

yozaam committed:
Fixes #10446, shorten "out of element" message

Based on the suggestion this has been made into  "element end"

leonardder commented:
@yozaam Out of interest, how did you manage in asking me for review? I thought this was only for collaborators.

yozaam commented:
> @yozaam Out of interest, how did you manage in asking me for review? I thought this was only for collaborators.

I'm sorry, when sending the request I just got the button on the right to ask for a reviewer, and since you opened the issue I though I'll ask you

yozaam committed:
Solving Flake8 errors

>     * [Build (for testing PR)](https://ci.appveyor.com/api/buildjobs/t4jxxbylk71fuisy/artifacts/output/nvda_snapshot_pr10714-19549,15bfdb32.exe)
> 
>     * PR introduces Flake8 errors
> 
> 
> See [test results](https://ci.appveyor.com/project/NVAccess/nvda/builds/30310576/tests) for [failed build](https://ci.appveyor.com/project/NVAccess/nvda/builds/30310576) of [commit 15bfdb32b4](https://github.com/nvaccess/nvda/commit/15bfdb32b4)

leonardder commented:
> I'm sorry, when sending the request I just got the button on the right to ask for a reviewer, and since you opened the issue I thought I'll ask you

No problem at all. I'm just amazed that you were able to see this button.

leonardder commented on source/speech/__init__.py:
```diff
@@ -1696,7 +1696,7 @@ def getControlFieldSpeech(  # noqa: C901
 			joinedRoleText = " ".join(roleTextSequence)
 			out = [
 				# Translators: Indicates end of something (example output: at the end of a list, speaks out of list).
```
Please fix this comment accordingly

yozaam replied:
Yes I will do this ASAP, thank you

LeonarddeR reviewed (changes requested):
There are several additional `out of` messages, please use `git grep "_(\"out of "` to find them

yozaam committed:
"element end" messages

Adriani90 commented:
@yozaam you should fix the flake8 errors otherwise this PR cannot be merged.
1. Go to test results
2. Go to artefacts
3. Open flake8errors.txt where the errors are displayed along with the source file
4. solve them in the source file.
In your case it seems there are just white space errors before and after the = operator in a function.

yozaam commented:
> @yozaam you should fix the flake8 errors otherwise this PR cannot be merged.
> 
>     1. Go to test results
> 
>     2. Go to artefacts
> 
>     3. Open flake8errors.txt where the errors are displayed along with the source file
> 
>     4. solve them in the source file.
>        In your case it seems there are just white space errors before and after the = operator in a function.

Thank you so much for explaining it to me in this format :D I am fixing it right now

yozaam committed:
"element end" flake8 errors

yozaam committed:
"element end" message flake8 errors

feerrenrut reviewed (changes requested):
Apologies this has been pending for so long, and welcome to the NVDA project 🎉

As I [mentioned on the issue](https://github.com/nvaccess/nvda/issues/10446#issuecomment-773852041) I'm happy let our alpha users try this out and give feedback. 

Though using `git grep -i "_(\"out of" -- "*.py"` I found one more:
`source/appModules/kindle.py:                _("out of popular highlight")`

Fix this up and we can go ahead.

CyrilleB79 commented:
Hello
Sorry to comment here a bit late.

Here is a strange use case that we should take into consideration.
Steps to reproduce:
* On a web page, press L to go to first element of a list
* Press UpArrow
In current version of NVDA, you hear "Out of list".
With this PR, if I understand correctly, you would hear "List end". This seems strange because we go out of the list across its start boundary.

Any feedback from braille users on this point?

For real containers (tables, lists, etc.), I do not have any better suggestion than the existing "out of". For containers that seem more formatting (e.g. spelling error, link), I guess "no spelling error" / "no link" would be suitable, but maybe it's out of the scope of this PR.

I do not advocate for any specific solution or change. But I wanted at least highlight this use case.

I do not

feerrenrut commented:
Thanks @CyrilleB79, that's a very useful observation. I'm not sure what to suggest either. I'm also interested in braille users perspective on this. For those not familiar with the braille behavior the start of a list shows `lst` and the end shows `lst end`. If going through the document backwards, you encounter `lst end` first then `lst`.

seanbudd commented:
Closed as this has been taken over by the more recently maintained and created #12260
